### PR TITLE
update go.mod & main.go errors

### PR DESCRIPTION
**Pull request by kikisdeliveryservice**
Update: So I think this go.mod hadnt been updated since it moved from andrew's repo over here.  

so I made a fresh go.mod, fixed some import errors in main.go and fixed a go fmt missing operand.

also renamed the (incorrectly named) ign2to3 to ignconverter

### CODE REVIEW

**Comment by arithx** [comment]
Notably the top level files do specify the package name as `ign2to3`, is this tested?

**Comment by kikisdeliveryservice** [comment]
Just to clarify:

importing `github.com/coreos/ign-converter` :
```
$ make go-deps
go mod tidy
go: finding github.com/coreos/ign-converter latest
go: github.com/coreos/ign-converter@v0.0.0-20200117071129-cfed8ca7a6b4: parsing go.mod: unexpected module path "github.com/coreos/ign2to3"
go: error loading module requirements
make: *** [Makefile:60: go-deps] Error 1
```
No problem so I import `github.com/coreos/ign2to3` : 
I get no changes whatsoever to my go.sum (even tho i should!!)

importing `github.com/coreos/ign-converter/ign2to3 : pulls in some stuff/updates my go.sum

I think that this is a relic from when the repo was called ign2to3? if module path was correct as `github.com/coreos/ign2to3` then should be able to import using that..

@bgilbert @jlebon sanity check please? go.mod is confusing

**Comment by jlebon** [comment]
I'm not a golang expert by any means, but yeah it'd be less confusing to just match the package name with the module name and the repo name.

**Comment by kikisdeliveryservice** [comment]
Ok so i redid the go.mod file which hadn't been updated in a long time.  I'm unsure what this main.go function is, but it had some problems evening running go build ie import issues and golang error, which i also fixed.

File: internal/main.go [hunk 1]
```diff
@@ -22,9 +22,7 @@ import (
 	"os"
 	"strings"
 
-	"github.com/coreos/ign2to3"
```
Comment (arithx) [inline internal/main.go]: It looks like there are things that are still using stuff in the `ign2to3` package in the file even after this change.

I'm not sure what the correct answer is to import it though (maybe `github.com/coreos/ign-converter/ign2to3` or just `github.com/coreos/ign-converter`?)

**Comment by kikisdeliveryservice** [comment]
cc: @yuqi-zhang

File: internal/main.go (hunk 1 above)
Comment (kikisdeliveryservice) [reply internal/main.go]: gahhh let me fix again.. (I swear i did this already)

Comment (kikisdeliveryservice) [reply internal/main.go]: ok done

File: internal/main.go [hunk 2]
```diff
@@ -22,9 +22,9 @@ import (
...
+	ign2to3 "github.com/coreos/ign-converter/ign2to3"
```
Comment (arithx) [inline internal/main.go]: Why the alias? The package is already `ign2to3`

File: internal/main.go [hunk 3]
```diff
@@ -112,8 +112,8 @@ func main() {
 		// translate from 2 to 3
 		mapping := getMapping(fsMap)
 
-		// parse to 2.4.0-experimental
-		cfg, rpt, err := v2_4_experimental.Parse(dataIn)
+		// parse
+		cfg, rpt, err := v2_3.Parse(dataIn)
```
Comment (arithx) [inline internal/main.go]: This technically is a potentially breaking change if someone was using a field that is present in `2.4.0-experimental` but not `2.3.0`; might make sense to switch to either `2.4.0` or `2.5.0-experimental` (with Ignition `0.35.0`)

Comment (kikisdeliveryservice) [reply internal/main.go]: I dont think this is a breaking change bc main.go in master currently does not work at all.

It gives error:
`internal/main.go:122:35: cannot use cfg (type "github.com/coreos/ignition/config/v2_4_experimental/types".Config) as type "github.com/coreos/ignition/config/v2_3/types".Config in argument to ign2to3.Translate`

With my change it actually works now.

Comment (arithx) [reply internal/main.go]: Ah, I see now that the actual `Translate` function is expecting a `v2_3` config.

**Pull request merged**